internal/objects/service/service.go

```diff
@@ -83,7 +81,7 @@ var (
 	//  https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
 	//
 	LbAnnotations = map[operatorv1alpha1.LoadBalancerProviderType]map[string]string{
-		awsProviderType: {
+		operatorv1alpha1.AWSLoadBalancerProvider: {
 			awsLbBackendProtoAnnotation: "tcp",
```

Comment:
I believe this annotation should only be included if Classic type of Load Balancer is specified (https://projectcontour.io/guides/deploy-aws-nlb/)